### ISSUE TITLE
Why loss function value is too high? Is it expecteted result?

### ISSUE THREAD

**Issue by padeirocarlos** [hustvl/PD-Quant#7]
I tried running your code for  with a pre-trained ResNet50 and MobilieNetV2 model. I got loss function value for  output and  pred losses:
     
rec_loss = lp_loss(pred, tgt, p=self.p)
        :param pred: output from quantized model
        :param tgt: output from FP model
[https://github.com/hustvl/PD-Quant/blob/main/quant/layer_recon.py#L196](url)

pd_loss = self.pd_loss(F.log_softmax(output / self.T, dim=1), F.softmax(output_fp / self.T, dim=1)) / self.lam
        :param output: prediction from quantized model
        :param output_fp: prediction from FP model
[https://github.com/hustvl/PD-Quant/blob/main/quant/layer_recon.py#L200](url)
 
Are there additional settings I missed?

**Comment by jiawei-liu1103** [comment]
The loss value of different blocks is different, and the loss value of some blocks may be high which is caused by the accumulation of quantization errors layer by layer. What are the specifics of your loss?

**Comment by padeirocarlos** [comment]
How do you deal with accumulation caused by quantization errors layer by layer? If am not wrong your manage the quantization block by block not entire model! I can see this in your code?!

**Comment by jiawei-liu1103** [comment]
The accumulation of errors in our code is unavoidable, so the high block loss value is expected.
